tests/python/test_relational_vcfs.py

```diff
@@ -0,0 +1,90 @@
+import nose
+
+import tempfile
+
+
+
+def test_vcf_to_csv():
+    import vcf, csv
+    from common.relational_vcf import vcf_to_csv
+    vcfreader = vcf.Reader(open('tests/data/snv.vcf'))
+    columns = ['contig', 'position', 'sample:DP', 'info:VAF', 'info:DP', 'something']
+    filename = vcf_to_csv(vcfreader, columns, None,
+                          default_values={'something': 'Something!'})
+
+    with open(filename) as fd:
+        rows = list(csv.reader(fd))
+
+    # first two are from the same sample
+    assert rows[0] == ['20', '61795', '44', '', '81', 'Something!']
+    assert rows[1] == ['20', '61795', '37', '', '81', 'Something!']
+    # this is the last sample
+    assert rows[-1] == ['20', '75254', '40', '', '74', 'Something!']
+    assert len(rows) == 20 # 10 records, 2 samples each = 20 rows
+
+
+def test_csv_to_vcf():
```

Comment:
add a comment indicating that this is testing that `vcf_to_csv` and `csv_to_vcf` are inverses—it's actually testing more than just `csv_to_vcf` (it would be doing just that if you checked in a CSV file to tests/data, for example, rather than generating it)